playbook.md

```diff
@@ -673,6 +673,68 @@ Findings and recommendations are not the same. Findings mean that ‘we know thi
 **10. Refine outputs and handover**
 These are things like slide decks, user journeys and user needs. We start drafting these early, iterating and adding to them as we go along. We share them frequently ‘just as they are’ so that clients have the opportunity to give us feedback without us spending lots of time making them pixel-perfect. This means we often need to refine part-finished, paper versions of outputs before handing them over to the client as final deliverables. User researchers do the bulk of this work, but it's a team effort. We allocate time before the end of the project to do this and bring in a designer if needed.
 
+### Product management
+
+#### Product management principles
+
+**Set the destination, don’t chart the course**
+Product managers help empower multidisciplinary teams by refining and articulating the problem the team is trying to solve, and the outcomes they’re trying to produce. 
+
+Communicating the vision and goals for a project through a clear and well-maintained roadmap is a core part of product management. 
+
+**Define and defend scope**
+Getting agreement from everyone involved is crucial to successful delivery. 
+
+Product managers should set the direction at the start of a project, and continue to articulate, communicate and defend the scope throughout, so that the team can focus on delivering. 
+
+**Convert insight into action**
+User research helps inform our objectives, goals, and priorities by telling us more about the problems users are having with the current state, or the needs they have for something different. 
+
+Product managers need to be able to balance consider user research insights with constraints like time, capacity, and strategic objectives to prioritise the next most important thing. 
```

Comment:
"Product managers need to be able to balance consider" I think one of the verbs is redundant, or needs to be a noun?